dashboard-for-desktop/articles/dashboard-designer/designing-dashboard-items/pivot/conditional-formatting.md

```diff
@@ -4,45 +4,90 @@ author: Natalia Kazakova
+	* [Gradient Ranges](../../appearance-customization/conditional-formatting/gradient-ranges.md)
+	* [Bar](../../appearance-customization/conditional-formatting/bar.md) 
+	* [Bar Color Ranges](../../appearance-customization/conditional-formatting/bar-color-ranges.md) 
+	* [Bar Gradient Ranges](../../appearance-customization/conditional-formatting/bar-gradient-ranges.md) 
 
-![GreaterThanDialog_Pivot](../../../../images/img118680.png)
+## Create and Edit a Format Rule
 
-This dialog contains the following settings specific to Pivot.
-* **Intersection mode** specifies the level on which to apply conditional formatting to pivot cells. The following levels are supported.
-	1. _Auto_ - Identifies the default level. For the Pivot dashboard item, _Auto_ identifies the _First Level_.
-	2. _First Level_ -  First level values are used to apply conditional formatting.
-	3. _Last Level_ -  The last level values are used to apply conditional formatting.
-	4. _All Levels_ -  All pivot data cells are used to apply conditional formatting.
-	5. _Specific Level_ - Values from the specific level are used to apply conditional formatting.
-* If you specified the **Intersection mode** as _Specific Level_, use the **Row dimension** and **Column dimension** combo boxes to set the specific level.
-* The **Apply to row** and **Apply to column** check boxes allow you to specify whether to apply the formatting to the entire pivot row/column.
+You can create and edit format rules in the following ways:
 
-> [!NOTE]
-> If you are creating a new format rule for the dimension from the **Columns/Rows** section, the corresponding format condition dialog would not contain any Pivot specific settings.
+* Click the **Edit Rules** button on the **Home** ribbon tab. 
+
+* Click the measure/dimension menu button in the Data Item's pane and select **Add Format Rule**/**Edit Rules**. 
+
+Refer to the following topic for information on how to create and edit format rules: [Conditional Formatting Common](../../appearance-customization/conditional-formatting.md).
+
+## Pivot-Specific Format Condition Settings
+
+You can configure and customize current format condition appearance settings.
+
+* Choose a predefined background color/font or click an empty square to add a new preset in the **Appearance** tab.
+
+	![NewRuleDialog_AppearanceTab](../../../../images/img118585.png)
+
+* Add a predefined icon in the **Icons** tab.	 
+
+The **Appearance** tab contains the following Pivot-specific settings:
+
+| Option | Description |
+| --|--|
+| **Enabled** | Enables/ Disables the current format rule. |
+| **Intersection Mode** | Specifies the level on which to apply conditional formatting to pivot cells. |  
+| **Intersection Row/Column Dimension**  | Applies the format rule to the specified row/column dimension, if you select the _Specific Level_ as the intersection mode.|
+| **Apply to Row/Column** | Specifies whether to apply the formatting to the entire Pivot's row/column.|
```

Comment:
Do you mean:

`Specifies whether to apply the formatting to the Pivot item's entire row/column.`

Reply:
Thanks. Please apply the change.